Multisig/Logic/Ledger/LedgerController.swift

```diff
@@ -30,14 +30,27 @@ class LedgerController {
         }
         let path = "44'/60'/0'/0/\(index)"
```

Comment:
do you account for different derivation paths? (ledger and ledger live)

Reply:
will be handled in future PRs